lib/components/spinner/index.tsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import { CircularProgress } from '@material-ui/core';
-import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
-const Spinner = ({ isWhite, ...props }) => {
+type OwnProps = typeof CircularProgress & {
+  isWhite: boolean;
+};
+
+const Spinner: FunctionComponent<OwnProps> = ({ isWhite, ...props }) => {
```

Comment:
we might as well get rid of the `...props` ambiguity at the same time…

```ts
const Spinner = ({
  isWhite,
  ...props
}: CircularProgressProps & { isWhite: boolean }) => {
```

Reply:
Doesn't what I have do that? I like what you have better but want to know what doesn't work with what I had.

Reply:
maybe because I missed what you had before? I didn't notice the `CircularPropgressProps` in the first line of the type definition 😉